bin/setup.js

```
@@ -1,64 +1,86 @@
+
+      case 'review':
+        await executeReview(PROJECT_ROOT, {
+          detailed: parsed.options.detailed,
+          json: parsed.options.json,
+          fix: parsed.options.fix,
+          noCodexGuide: parsed.options.noCodexGuide,
+        });
+        break;
+
+      case 'commit-todo':
+        const subCommand = parsed.commandArgs[0] || 'enforce';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)